client/components/Layout/Images/BlurhashImage.vue

```diff
@@ -1,19 +1,19 @@
 <template>
-  <div class="absolute">
-    <div v-if="!error" ref="img" class="absolute">
-      <blurhash-canvas
-        v-if="hash"
-        key="canvas"
-        :hash="hash"
-        :width="width"
-        :height="height"
-        :punch="punch"
-        class="absolute"
-      />
+  <div ref="imageElement">
```

Comment:
Shouldn't we use better the ``img`` element to pass it to the function?

Reply:
After testing what I comment on https://github.com/jellyfin/jellyfin-vue/pull/769#discussion_r608576319 both in your branch and in master, I see that the Cast & Crew images have a far superior resolution in your branch than in master, which makes me suspect that I was indeed right about this change.

Maybe, being the child absolute positioned, the size can't be determined correctly and it's loading full resolution?

Reply:
That wasn't working, probably because the image doesn't contain anything and is absolute.

This element has the same size anyway, and is technically our image component.